backend/Tournament/exceptions.py

```diff
@@ -0,0 +1,13 @@
+from rest_framework.exceptions import APIException
+from rest_framework import status
+
+
+class TournamentSetupException(APIException):
+    status_code = status.HTTP_403_FORBIDDEN
+    default_detail = 'There was an error creating the tournament or its players.'
+    default_code = 'tournament_creation_error'
+
+class TournamentInProgressException(APIException):
+    status_code = status.HTTP_403_FORBIDDEN
```

Comment:
Why are these using the forbidden status? that is an exception for when a resource that the requesting user is not allowed to access is used - which is a very specific case. Bad Request is usually the default one. please refer to the documentation about the status codes: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#server_error_responses.